frontend/src/components/search/SearchResultList.vue

```diff
@@ -336,6 +339,18 @@ export default class SearchResultList extends Mixins(SearchApiRequestMixin) {
   private getLookupValueInfrastruktureinrichtung(key: string, list: Array<LookupEntryDto>): string | undefined {
     return !_.isUndefined(list) ? list.find((lookupEntry: LookupEntryDto) => lookupEntry.key === key)?.value : "";
   }
+
```

Comment:
Kurzform:

` public disableAbfrageCard(item: AbfrageSearchResultDto): boolean {
    return this.hasOnlyRoleAnwender()
      ? item.statusAbfrage != StatusAbfrage.ErledigtMitFachreferat &&
          item.statusAbfrage != StatusAbfrage.ErledigtOhneFachreferat
      : false;
  }`